src/main/java/org/fcrepo/spec/testsuite/authz/WebACModes.java

```diff
@@ -539,4 +539,90 @@ public void controlDisallowedPUT(final String uri) {
         childAclResponse.then().statusCode(403);
     }
 
+    /**
+     * 5.7.1-A acl:Append for LDP-RS MUST test conditions
+     *
+     * @param uri of base container of Fedora server
+     */
+    @Test(groups = { "MUST" })
+    @Parameters({ "param1" })
+    public void appendNotWriteLdpRsMust(final String uri) {
+        final TestInfo info = setupTest("5.7.1-A", "appendNotWriteLdpRsMust",
+                "When a client has acl:Append but not acl:Write for an LDP-RS they MUST " +
+                        "not DELETE, not PATCH that deletes triples, not PUT on the resource, and " +
+                        "must be able to PUT a child resource",
+                "https://fedora.info/2018/06/25/spec/#append-ldprs", ps);
+
+        final String resourceUri = createResource(uri, info.getId());
+        createAclForResource(resourceUri, "user-read-append.ttl", this.username);
+
+        // perform DELETE to resource as non-admin
+        final Response deleteResponse = doDeleteUnverified(resourceUri, false);
+        // verify failure.
+        deleteResponse.then().statusCode(403);
+
+        // perform PUT to resource as non-admin
+        final Response putResponse =
+                doPutUnverified(resourceUri, new Headers(new Header("Content-Type", "text/plain")),
+                        "test", false);
+        // verify failure.
+        putResponse.then().statusCode(403);
+
+        // perform PUT to child resource as non-admin succeeds.
+        final Response putChildResponse =
```

Comment:
It may be best to break this case off into a separate test given "if the implementation supports creating resources using PUT" part, it is valid for this case to produce something other than a 201 if PUT is not supported